locations/spiders/osservaprezzi_carburanti_it.py

```diff
@@ -0,0 +1,63 @@
+import json
+
+from scrapy import Spider
+from scrapy.http import JsonRequest, Response
+
+from locations.categories import Categories, Fuel, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.geo import point_locations
+
+
+class OsservaprezziCarburantiITSpider(Spider):
+    name = "osservaprezzi_carburanti_it"
+
+    FUELS = {
+        "Gasolio": Fuel.DIESEL,
+        "GPL": Fuel.LPG,
+        "Metano": Fuel.CNG,
+        # "Blue Diesel": Fuel.ADBLUE, ?
+    }
+
+    BRANDS = {
+        "Api-Ip": {"brand": "IP", "brand_wikidata": "Q646807"},
+        "Esso": {"brand": "Esso", "brand_wikidata": "Q867662"},
+        "Q8": {"brand": "Q8", "brand_wikidata": "Q1634762"},
+        "Tamoil": {"brand": "Tamoil", "brand_wikidata": "Q706793"},
+        "AgipEni": {"brand": "Eni", "brand_wikidata": "Q565594"},
+        "Giap": {"brand": "Giap", "brand_wikidata": None, "nsi_id": "giap-e8f712"},
+        "Energas": {"brand": "Energas", "brand_wikidata": None, "nsi_id": "energas-e8f712"},
+        "Keropetrol": {"brand": "Keropetrol", "brand_wikidata": None, "nsi_id": "keropetrol-e8f712"},
+        "San Marco Petroli": {"brand": "San Marco Petroli", "brand_wikidata": None, "nsi_id": "sanmarcopetroli-e8f712"},
```

Comment:
```suggestion
        "Giap": {"brand": "Giap", "brand_wikidata": None},
        "Energas": {"brand": "Energas", "brand_wikidata": None},
        "Keropetrol": {"brand": "Keropetrol", "brand_wikidata": None},
        "San Marco Petroli": {"brand": "San Marco Petroli", "brand_wikidata": None},
```

NSI ids aren't stable, we don't hard code them. The real solution to them being missing from the output, it to add wikidata to NSI (and here).

Reply:
At least I'll learn how to edit NSI 😄

Thanks for the feedback.

Reply:
From what I

Reply:
I'm happy to do it if you want, but I expect the reason they don't have wikidata in NSI is because they don't have wikidata... So you'd be learning how to edit wikidata as well :)

I'll can do them tomorrow if you want.

Reply:
I can do it, if you don't mind it taking a bit longer. There's going to be more brands in the data that are not yet in the spider.

Reply:
No rush, take as long as you like. I'd probably only do the brands with more than 100 or 200 entries, there are a lot of brands with less than 50 pois in this dataset.

```json
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Ego": 210,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/SanMarcoPetroli": 215,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Costantin": 232,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Retitalia": 233,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Beyfin": 251,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Shell": 311,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/KEROPETROL": 312,
  "atp/gov_osservaprezzi_carburanti_it/unmapped_brand/Europam": 432,
```
`$ cat osservaprezzi_carburanti_it.json  | jq '[to_entries | .[] | select(.key | startswith("atp/gov_osservaprezzi_carburanti_it/unmapped_brand"))] | sort_by(.value) | from_entries'`